### PR TITLE
Creates fromUnix function to return with applied timezone

### PR DESCRIPTION
**Pull request by Caps-Looking**
Fix https://github.com/korlibs/klock/issues/158

### CODE REVIEW

File: klock/src/commonTest/kotlin/com/soywiz/klock/TimeFormatTest.kt [hunk 1]
```diff
@@ -53,4 +53,15 @@ class TimeFormatTest {
         assertEquals(59, time.second)
         assertEquals(0, time.millisecond)
     }
+
+    @Test
+    fun testFromUnix() {
```
Comment (soywiz) [inline klock/src/commonTest/kotlin/com/soywiz/klock/TimeFormatTest.kt:58]: 👍 no needed to mock the timezone in the end right?

File: klock/src/commonMain/kotlin/com/soywiz/klock/DateTimeTz.kt [hunk 2]
```diff
@@ -24,6 +24,12 @@ class DateTimeTz private constructor(
         /** Creates a new local [DateTimeTz] from a [unix] time */
         fun fromUnixLocal(unix: Double): DateTimeTz = DateTime(unix).localUnadjusted
```
Comment (soywiz) [inline klock/src/commonMain/kotlin/com/soywiz/klock/DateTimeTz.kt:25]: Do you think it makes sense to deprecate this? Could it be confusing to other people?

Comment (Caps-Looking) [reply klock/src/commonMain/kotlin/com/soywiz/klock/DateTimeTz.kt:25]: Absolutely!

**Comment by soywiz** [comment]
Sorry for the huge delay. I totally forgot about this. Thanks!

**Pull request merged**